src/modules/visitor-space/views/VisitorSpacesHomePage.tsx

```diff
@@ -33,10 +35,20 @@ export const VisitorSpacesHomePage: FC<DefaultSeoInfo> = (props) => {
 			if (showLinkedSpaceAsHomepage && linkedSpaceSlug) {
 				return <Loading fullscreen owner="root page logged" />;
 			}
-			return <LoggedInVisitorSpacesHome {...props} />;
+			return <LoggedInVisitorSpacesHome />;
 		}
-		return <LoggedOutVisitorSpacesHome {...props} />;
+		return <LoggedOutVisitorSpacesHome />;
 	};
 
-	return <VisitorLayout>{renderPageContent()}</VisitorLayout>;
+	return (
+		<VisitorLayout>
+			{renderOgTags(
+				tText('Bezoek pagina titel'),
+				tText('Bezoek pagina beschrijving'),
```

Comment:
Missing translations

Reply:
We'll extract these when we run through the deploy checklist